interface/modules/custom_modules/oe-module-claimrev-connect/src/Eligibility.php

```diff
@@ -0,0 +1,237 @@
+<?php 
+use OpenEMR\Modules\ClaimRevConnector\EligibilityData;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityInquiryRequest;
+use OpenEMR\Modules\ClaimRevConnector\SubscriberPatientEligibilityRequest;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityObjectCreator;
+use OpenEMR\Modules\ClaimRevConnector\ValueMapping;
+
+
```

Comment:
would die if $pid is null

Reply:
I tested this making $pid = null, while it didn't look horrible, I went ahead and added a message and an exit;